tensorflow/stream_executor/rocm/rocm_driver.cc

```diff
@@ -491,8 +491,8 @@ ROCMDriver::DeviceGetSharedMemConfig(int device_ordinal) {
   unsigned char valueC = static_cast<unsigned char>(value);
   uint32_t value32 = (valueC << 24) | (valueC << 16) | (valueC << 8) | (valueC) ;
   assert (value32 == value); // if mismatch this indicates case where hipMemsetAsyc can't emulate hipMemSetD32
-  hipError_t res =
-      hipMemsetAsync(pointer, value, uint32_count*4, stream);
+ hipError_t res =
```

Comment:
nitpicking: please fix indentation

Reply:
hmm..Dont see it locally..let me redit the line..